readthedocs/projects/constants.py

```diff
@@ -80,7 +80,10 @@
 
 PRIVACY_CHOICES = (
     (PUBLIC, _('Public')),
-    (PROTECTED, _('Protected')),
+    # Remove Protected for now since it cause confusions to users.
+    # There is a better way to manage this by using Version states
+    # See: https://github.com/rtfd/readthedocs.org/issues/5321
+    # (PROTECTED, _('Protected')),
```

Comment:
What do people with `Protected` in the DB currently see for this listing?

Reply:
Weird stuffs :)

Some places it will say `protected` (version listing) and some other `Public` (version details).

It may be better to remove it from the Form itself by overriding the field instead of from this global constant.

Reply:
Yeah, i'd have the same reservations around just removing it from the database full stop. I think we might need to be more gentle here, or perhaps even use a feature flag to grandfather projects into `protected`.

We also discussed maybe researching how projects are using the feature now, and probably messaging the users before making a hard change like this one.

Reply:
I'm not too worry about asking users how they use `protected` because it's definitely a weird concept and it's currently broken.

I don't want to remove it from the DB either, I just want to hide it from the users so they can not use `Protected` anymore in new projects/versions and get confused with it. The old ones should remain as they are: "in a Protected privacy level, but broken"